packages/medusa/src/strategies/batch-jobs/product/import.ts

```diff
@@ -565,6 +616,10 @@ class ProductImportStrategy extends AbstractBatchJobStrategy {
       return
     }
 
+    const isMedusaV2Enabled = this.featureFlagRouter_.isFeatureEnabled(
```

Comment:
todo: should we move that on the class level to be accessible everywhere? like a getter for example